plugins/woocommerce/client/blocks/tests/e2e/tests/accordion/accordion.block_theme.spec.ts

```diff
@@ -78,7 +78,6 @@ test.describe( `${ blockData.slug } Block`, () => {
 			frontendUtils,
 			pageObject,
 		} ) => {
-			await editor.insertBlock( { name: blockData.slug } );
```

Comment:
why was this removed?

Reply:
The block was added twice. Here and within [`insertAccordionGroup`](https://github.com/woocommerce/woocommerce/blob/dec4c3dce69c5d16bfbb020f2e1e29af7adfe3c8/plugins/woocommerce/client/blocks/tests/e2e/tests/accordion/accordion.page.ts#L83C6-L85C6) which is called in next line. So there were two Accordion Groups but only one was used in the test. I assume it was accidentally passing before.